state/protocol/badger/validity.go

```diff
@@ -110,3 +119,57 @@ func validCommit(commit *flow.EpochCommit, setup *flow.EpochSetup) error {
 
 	return nil
 }
+
+// validRootSnapshot checks internal consistency of root state snapshot
+func validRootSnapshot(snap protocol.Snapshot) error {
```

Comment:
```suggestion
func isValidRootSnapshot(snap protocol.Snapshot) error {
```

Reply:
👍 Updated in https://github.com/onflow/flow-go/pull/374/commits/33dd989b6c6c34a3aeb9d4abb249fe603fe2f293